com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java

```diff
@@ -57,6 +62,22 @@ public static boolean isDirectory(String fileName) {
         return fileName.endsWith("/");
     }
 
+    public static void cleanUp(Path filePath, Logger logger) {
```

Comment:
Didn't we have this function elsewhere?

Reply:
Yes, I moved it here because it was copy-pasted.